purchase_hide_report_print_menu/purchase.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# © 2016 Chafique DELLI @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api
+
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    @api.model
+    def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
+                        submenu=False):
+        context = self._context
```

Comment:
pas besoin de déclarer une variable, on peut simplement tester 'purchase_order' in self._context ou  self._context.get('purchase_order') qui me semble plus propre